docs/loki-deployment.md

```diff
@@ -0,0 +1,161 @@
+# Overview
+This page lists the 2 common Loki deployments, to help user to better understand Loki architecture.
+
+Loki consists several components as below:
+![Loki Arch](loki_arch.png)
```

Comment:
I know this from a presentation slide, but could somebody verify that the querier is actually talking to the ingester?

Reply:
yes it is. From my testing, I push logs through API to distributor vip, then distributor uses gRPC to send logs to Ingester.

Reply:
That's Distributor and Ingester... Did you see any direct Querier to Ingester communication?

Reply:
yes there is @sh0rez

Reply:
sorry, man. Yes, for querier, it definitely communicates with ingester since it will loads data in Ingester memory (which has not flushed to Storage) to get recent data.

Reply:
Okay, then I didn't say anything ;)